src/react.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import './polyfill';
+import { importWithDependencies, importDependenciesOf } from './index';
 
 class ExternalComponent extends Component {
```

Comment:
Open to suggestions here, also wanna move to Hooks and @dai_shi react async hooks